File: languages/java/testData/variable/notVariable.java
package testData.variable;

public class MyClas<caret>s {
    public int myField = 0;
    public static int collision = 1;

    public void setF<caret>ield(int boi) {
        myField = boi;
    }

    public static void main(String[] args) {
        MyClass instance = new MyClass();
        instance.myField = 1;
    }
}

File: languages/kotlin/testData/variable/notVariable.kt
package variable

class MyClass<caret>2(not_collision: Boolean, val collision1: Int = 0) {
    var myField = 0
    private val collision2 = 1
    fun setFi<caret>eld(boi: Int) {
        myField = boi
    }

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val instance = MyClass2(true)
            instance.myField = 1
        }
    }
}
